service/main.tf

```diff
@@ -1,5 +1,9 @@
 #tfsec:ignore:aws-vpc-no-public-ingress-sgr tfsec:ignore:aws-vpc-no-public-egress-sgr
 module "ecs_alb_service_task" {
+  #checkov:skip=CKV_AWS_249:Roles are passed in
+  #checkov:skip=CKV_AWS_97:Transit encryption is optional and can be enabled by passing in configuration to the cloudposse module
```

Comment:
transit encryption can be enabled, but it does not look like you are enabling it in the example below

Reply:
Ultimately, I feel this would be down to the consumer of the module - if we were to enable this, we would _have_ to also forcibly provision at least one efs volume per service, even if it wouldn't be needed.

Volumes are passed into the module with:
```terraform
efs_volumes        = var.efs_volumes
```
type def:
```terraform
type = list(object({
    host_path = string
    name      = string
    efs_volume_configuration = list(object({
      file_system_id          = string
      root_directory          = string
      transit_encryption      = string
      transit_encryption_port = string
      authorization_config = list(object({
        access_point_id = string
        iam             = string
      }))
    }))
  }))
```
and transit encryption can be by setting transit encryption to `'true'`.

Reply:
yes, it is up to the consumer, however, I  feel this should be enabled by default (transit_encryption  set to true) with an option to be overwritten

Reply:
Valid point and taken on board - I have set the type def to be optional, but defaulted to `"enabled"`

https://github.com/ministryofjustice/modernisation-platform-terraform-ecs-cluster/blob/57208030e1b25777dec64a82da306b0622644c3a/service/variables.tf#L123

Reply:
brilliant, thanks for addressing it George! :)